src/status_im/ui/components/contact/contact.cljs

```diff
@@ -27,27 +27,28 @@
        [text {:style st/info-text}
         info])]]))
 
-(defn contact-view [{:keys [contact extended? on-press extend-options info]}]
+(defn contact-view [{:keys [contact extended? on-press extend-options info show-forward?]}]
   [touchable-highlight (when-not extended?
                          {:on-press (when on-press #(on-press contact))})
-   [view
     [view st/contact-container
      [contact-inner-view {:contact contact :info info}]
+     (when show-forward?
```

Comment:
Is `show-forward?` linked to the presence of `on-press`? As in, every time there is a `on-press` then forward should be show? Might be related to the use (or non-use) of `touchable-highlight`

Reply:
i think it means "next step", from UI pov, so if there is a forward  icon, it means you will not return to this screen after press, so `show-forward?` and `on-press` not related

Reply:
:+1: